css/main.css

```diff
@@ -325,3 +325,82 @@ li {
 .skills {
   list-style: inside url("../media/svg/circle.svg");
 }
+
+footer {
+  height: 500px;
+  margin-top: 3%;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 300;
+}
+
+.form-container {
+  /* display: flex;
+  justify-content: space-between;
+  width: 90%; */
+  margin-left: 5%;
+}
+
+.form-container p {
+  /* width: 35%; */
+  font-weight: lighter;
+  font-size: 33px;
+  margin-bottom: 33px;
+}
+
+.inputs {
+  width: 50%;
+  display: flex;
+  flex-direction: column;
+}
+
+.name-input {
+  display: flex;
+  width: 100%;
+  justify-content: space-between;
+}
+
+.first-name {
+  /* display: flex; */
```

Comment:
- [ ] Please kindly remember to display all your inputs with flexbox. 

![image](https://user-images.githubusercontent.com/33432289/136430317-b6afed1e-0242-440d-8c49-c7c57bccf03b.png)